packages/fxa-auth-server/lib/senders/emails/partials/cadReminder/index.ts

```diff
@@ -5,15 +5,16 @@
 import { appBadges, button, automatedEmailNoAction } from '../../partials';
 
 export const cadReminder = `
+  <mj-include path="./lib/senders/emails/css/global.css" type="css" css-inline="inline" />
   <mj-section>
     <mj-column>
-      <mj-text css-class="header-text"><span data-l10n-id="<%- templateName %>-title"><%- headerText %></span></mj-text>
-      <mj-image css-class="sync-logo" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
+      <mj-text css-class="text-header"><span data-l10n-id="<%- templateName %>-title"><%- headerText %></span></mj-text>
+      <mj-image css-class="graphic-devices" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
```

Comment:
Just realized this image is a little different than what we have in `fxa` layout so changed it's classname too